<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColliderComp</class>
 <widget class="QGroupBox" name="ColliderComp">
  <property name="windowTitle">
   <string>Collider</string>
  </property>
  <property name="title">
   <string>Collider</string>
  </property>
  <layout class="QFormLayout" name="colliderLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="shapeTypeLabel">
     <property name="text">
      <string>Shape type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="shapeType">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="shapeInfoLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
